### ISSUE TITLE
Bad representation binary keys

### ISSUE THREAD

**Issue by eskomorokhov** [badboy/rdb-rs#13]
Binary keys printed like 
db=3 ���YƠ�D�F
��[0] -> {eAB2J1IxJRAUZRkfHRAZEBwXGx8CCw==, score=0.000002}
And it is very hard to parse output. It will be better it will be shown as encoded binary data.

**Comment by badboy** [comment]
Yup, I did not yet decide how to properly present them.

**Comment by eskomorokhov** [comment]
I think it's not a problem to present such keys as byte escape format like "\xfff" or encode it to base64.
